__tests__/endpoints.test.js

```diff
@@ -44,4 +44,46 @@ describe("GET requests", () => {
         });
     });
   });
+  describe("/api/articles/:article_id", () => {
+    it("responds with a single article object", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+          expect(typeof response.body.article).toBe("object");
+        });
+    });
+    it("the returned object contains the correct properties", () => {
+      const article_id = 1;
+      return request(app)
+        .get(`/api/articles/${article_id}`)
+        .then(({ body }) => {
+          expect(body.article).toEqual({
```

Comment:
Just to note that using toEqual to test a response object like this may cause you problems in the future if features are added that add properties to the object (such as the comment_count ticket later on!)